scripts/verify-replace.sh

```diff
@@ -16,26 +20,57 @@ do
     echo                                                                     
     echo =========================================================================================                                            
     repo_path=${BASE_REPO_PATH}/$(basename ${repo})
+    ERRFILE=$(mktemp ${TMP_DIR}$(basename ${repo})-error.XXX)
+    STDOUTFILE=$(mktemp ${TMP_DIR}$(basename ${repo})-output.XXX)
     echo "Cloning repo in /tmp"
     git clone --depth=1 ${repo} ${repo_path}
     echo "Repo cloned successfully"
     cd ${repo_path}
     if ! make pre-verify; then
-        ERRORLIST+="($(basename ${repo}))"
+        ERRORREPOLIST+="$(basename ${repo}) "
         continue
     fi
     echo "Initiating 'go mod replace' of current toolchain common version in dependent repos"
     go mod edit -replace github.com/codeready-toolchain/toolchain-common=${C_PATH}
-    make verify-dependencies || ERRORLIST+="($(basename ${repo}))"
+    make verify-dependencies 2> ${ERRFILE} 1> ${STDOUTFILE}
+    rc=$?
+    if [ ${rc} -ne 0 ]; then
+    ERRORREPOLIST+="$(basename ${repo}) " 
+    ERRORFILELIST+="${ERRFILE}  "
+    STDOUTFILELIST="${STDOUTFILE} "
+    fi
     echo                                                          
     echo =========================================================================================
     echo                                                           
 done
-if [ ${#ERRORLIST[@]} -ne 0 ]; then
+echo                "Summary"
+if [ ${#ERRORREPOLIST[@]} -ne 0 ]; then
     echo "Below are the repos with error: "
-    for e in ${ERRORLIST[*]}
+    for e in ${ERRORREPOLIST[*]}
     do
-        echo "${e}"
+        for c in ${ERRORFILELIST[*]}
+        do
+            if [[ ${c} =~ ${e} ]]; then 
```

Comment:
not using just a single chars, bust whole words for the variable would significantly improve the reading of the code

Reply:
okay i agree, updated it to better names